src/libs/typesystem/SpecElektra/SpecElektra.cabal

```diff
@@ -0,0 +1,31 @@
+name:                SpecElektra
+version:             0.1.0.0
```

Comment:
A version schema with patch level? How many releases do you plan to do :smile: 

Or is this standard for cabal?

Reply:
yes thats the default version when you create a new cabal project. i'm not really sure which naming scheme makes sense for this so i just left the default for now. i thought about just using KDB_VERSION for everything but the downside is that if nothing haskell related happens during 2 releases its quite meaningless.

Reply:
For other systems like cabal we usually have a single number for the version of the binding itself and additionally Elektra's version number (Updates in Elektra could influence the behavior of the binding, too).

What about using: V.X.Y.Z where:

- V is your number where you indicate (incompatible) changes of your binding
- X.Y.Z is the Elektra version

Reply:
sounds consistent, i don't think the bindings will change too much once i have the whole project done. i'll stick to that for now. Should i apply this scheme to each haskell library i created? (typechecker, regexdispatcher, specelektra, spectranslator, libfa, libelektra-haskell)